packages/devtools_app/lib/src/shared/theme.dart

```diff
@@ -426,6 +426,9 @@ extension ThemeDataExtension on ThemeData {
         fontSize: defaultFontSize,
       );
 
+  TextStyle get subtleErrorTextStyle =>
+      textTheme.bodyText2!.copyWith(color: colorScheme.errorTextColor);
```

Comment:
Let's do `subtleTextStyle.copyWith(color: colorScheme.errorTextColor)` here so that if `subtleTextStyle` ever changes any attributes other than color, those changes will also be reflected in `subtleErrorTextStyle`